assets/js/util.js

```diff
@@ -119,6 +119,25 @@ $(document).ready(function () {
     $('.modal').on('hide.bs.modal', function () {
         $('a[data-target=#' + $(this).attr('id') + ']').parents('li').removeClass('active');
     });
+
+    $(window).scroll(function () {
+        if($(window).scrollTop() > BACK_TO_TOP_ACTIVATION_HEIGHT) {
+            $('#backToTop').removeClass('hidden-xs hidden-sm hidden-md hidden-lg');
+            $('#backToTop').addClass('hidden-xs hidden-sm visible-md visible-lg');
```

Comment:
Basically, what I'm saying is that your code

```
$('#backToTop').removeClass('hidden-xs hidden-sm hidden-md hidden-lg');
$('#backToTop').addClass('hidden-xs hidden-sm visible-md visible-lg');
```

== 

```
$('#backToTop').removeClass('hidden-md hidden-lg');
$('#backToTop').addClass('hidden-xs hidden-sm visible-md visible-lg');
```

Does that make sense?

On the other hand, if you intend for a class to always be applied to a particular element, that should happen in the HTML, not the JS.